[Hieu] New functions in API system (in-depth check for images submission)

maybehieu opened the pull request:
Please review, currently quite junky code.

maybehieu committed:
[Hieu] New API test phase

maybehieu committed:
[PATCH] Remove unused import

maybehieu committed:
[PATCH] [Hieu] Hotfix for new API

maybehieu commented:
Incomplete

maybehieu commented:
Re-opening for review.

pewdspie24 commented on Data_Collector_API/PreProcesser.py:
```diff
@@ -0,0 +1,110 @@
+            bright = -1
+        else:
+            bright = 0
+        if blur == 1 and bright == 1:  # blurry and high-light
+            return -3
+        elif blur == 1 and bright == -1:  # blurry and low-light
+            return -4
+        elif blur == 1 and bright == 0:  # blurry
+            return -2
+        elif blur == 0 and bright == 1:  # high-light
+            return -1
+        elif blur == 0 and bright == -1:  # low-light
+            return 0
+        return 1
```
Make code DRY, too much elif

pewdspie24 commented on Data_Collector_API/PreProcesser.py:
```diff
@@ -0,0 +1,110 @@
+    blur = []
+    bright = []
+    ite = 0
+    processer = Processer()
```
Processor, I think?

pewdspie24 commented on Data_Collector_API/main.py:
```diff
@@ -52,6 +74,38 @@ def predict():
+    file.save(buffer)
+    image = np.array(Image.open(file, formats=["JPEG"]))
+    image = Image.fromarray(image)
+    processer = PreProcesser.Processer(byte_file)
```
Same thing here

pewdspie24 commented on Data_Collector_API/main.py:
```diff
@@ -52,6 +74,38 @@ def predict():
+        if os.path.exists(os.path.join(cwd, dir_path)):
+            time.sleep(1)
+            time_now = datetime.now().strftime("%Y%m%d_%H%M%S")
+            dir_path = os.path.join(str(0), time_now + ".jpg")
```
Is there a case that 2 or more images can be processed in 1 sec?

pewdspie24 reviewed (commented):
Pls check

maybehieu committed:
deleted:    Data_Collector_API/PreProcesser.py
	modified:   Data_Collector_API/PredictionPytorch.py
	modified:   Data_Collector_API/main.py

maybehieu committed:
[Hieu] Code refactoring

maybehieu committed:
[Hieu] Remove unused import

maybehieu commented:
I just pushed some changes in attempt of fixing the errors you've listed above, can you review it again?

pewdspie24 commented:
LGTM
